### PR TITLE
added tweet_search

### PR DESCRIPTION
**Pull request by k4u5h1k**
#### Resolves
Adds tweet_search module

#### Changes
- Added util/osint/twitter.py and modules/search/tweet_search.py

#### Checklist

- [X] I have read the Contribution & Best practices Guideline.
- [X] My branch is up-to-date with the Upstream `master` branch.
- [ ] The acceptance, integration, unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: maryam/core/util/osint/twitter.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+		self.framework = main.framework
+		self.q = q
+		self.max = limit
+		self._json = {}
+		self._tweets = []
+		self._verbose = verbose
+
+	def run_crawl(self):
+		self.framework.verbose('[TWEETSEARCH] Searching the twitter domain...')
+		self.header = {
```
Comment (saeeddhqan) [inline maryam/core/util/osint/twitter.py]: we can't use this forever

**Review by saeeddhqan — COMMENTED** [review]
use another name instead of twitter.

File: maryam/core/util/osint/twitter.py (hunk 1 above)
Comment (k4u5h1k) [reply maryam/core/util/osint/twitter.py]: so tweet_search.py?

Comment (k4u5h1k) [reply maryam/core/util/osint/twitter.py]: done

**Pull request merged**